### PR TITLE
[Blackbox-exporter] Add support for affinity and tolerations

### PR DESCRIPTION
**Pull request by ebabani**
#### What this PR does / why we need it:

Add support for affinities and tolerations to the prometheus blackbox exporter chart

#### Which issue this PR fixes
N/A
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/prometheus-blackbox-exporter/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 0.1.3
+version: 0.1.4
```
Comment (davidkarlsen) [inline stable/prometheus-blackbox-exporter/Chart.yaml]:
```suggestion
version: 0.2.0
```
new features

File: stable/prometheus-blackbox-exporter/templates/deployment.yaml [hunk 2]
```diff
@@ -30,6 +30,15 @@ spec:
       nodeSelector:
 {{ toYaml .Values.nodeSelector | indent 8 }}
     {{- end }}
+    {{- if .Values.affinity }}
+      affinity:
+{{ toYaml .Values.affinity | indent 8 }}
+    {{- end }}
+    {{- if .Values.tolerations }}
+      tolerations:
+{{ toYaml .Values.tolerations | indent 8 }}
```
Comment (davidkarlsen) [inline stable/prometheus-blackbox-exporter/templates/deployment.yaml]: indent 6 - since it is a list

**Comment by davidkarlsen** [comment]
@ebabani can you have a look at the comments please? TIA

**Comment by ebabani** [comment]
Updated, sorry about the delay.

**Pull request merged**